src/_scss/pages/search/results/table/_tableTypes.scss

```diff
@@ -54,6 +54,17 @@
                     background-color: $color-gold;    
                 }
             }
+              .tooltip-wrapper {
+                .tooltip__text {
+                    padding: 2rem 2rem 0 2rem;
```

Comment:
`rem()`?!?

Reply:
i thought we were moving away from that because it is part of bourbon